db/migrate/20160324174132_create_deployments.rb

```diff
@@ -0,0 +1,23 @@
+class CreateDeployments < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_deployments do |t|
+      t.string   :deployment_type
+      t.string   :deployment_version
+      t.boolean  :containerized, :default => true
+      t.string   :deployment_method
+      t.string   :deployed_provider_type
+      t.string   :deployed_provider_name
+      t.string   :ssh_private
+      t.string   :ssh_public
+      t.string   :ssh_user
+      t.string   :rhn_user
+      t.string   :rhn_pass
+      t.string   :rhn_sku
```

Comment:
It shouldn't matter much what the field names are called here, but in the UI we should reference rhsm (Red Hat Subscription Management) and not RHN.